README.md

```diff
@@ -93,12 +93,14 @@ Here're the steps to do it:
 1. Make sure `$GOPATH/bin` is in your `$PATH`, i.e., `export PATH=$GOPATH/bin:$PATH`.
 2. `bidbot init`.
 3. The output will ask you to sign a token with the owner address of your miner.
-4. Configure your _ask price_, other bid settings, and auction filters. See `bidbot help daemon` for details. You can edit the configuration file generated by `bidbot init` or use the equivalent flag for any given option.
-5. Use the signature you generated above to start the daemon: `bidbot daemon --miner-addr [address] --wallet-addr-sig [signature]`
+4. Configure your _ask price_, other bid settings, and auction filters. See `bidbot help daemon` for details. You can edit the configuration file generated by `bidbot init` (`~/.bidbot/config`) or use the equivalent flag for any given option.
+5. Use the signature you generated above to start the daemon as indicated by step 2. output.
```

Comment:
Prefer to avoid duplicating flags in the explained steps since this becomes stalled when doing changes. The deleted line 5. already was missing the needed flags for Lotus API addr and token. We already do the correct explanation in the `bidbot init` output, so better to let that be the source of truth.